### DISCUSSION TITLE
希望增加的功能

### DISCUSSION THREAD

**Discussion by qaz5391450** [SPlayer-Dev/SPlayer#157] [想法 Ideas]
1、新建本地歌单，可以不同步网易云。
2、批量操作。
场景：我想把周杰伦的所有歌都放到一个歌单里听，但目前必须是登录后才能创建。

**Comment by imsyy** [comment]
目前和网易云音乐api结合的太深了，等后面脱离api开发吧

**Reply by Takenforgranted** [reply]
> 目前和网易云音乐api结合的太深了，等后面脱离api开发吧

后续再添加别的音乐软件……不就成listen 1了？除非另起炉灶。

**Comment by qaz5391450** [comment]
支持大佬！
